homeassistant/components/manual/alarm_control_panel.py

```diff
@@ -385,7 +386,7 @@ def device_state_attributes(self):
         """Return the state attributes."""
         state_attr = {}
 
-        if self.state == STATE_ALARM_PENDING:
+        if self.state == STATE_ALARM_PENDING or self.state == STATE_ALARM_ARMING:
             state_attr[ATTR_PRE_PENDING_STATE] = self._previous_state
             state_attr[ATTR_POST_PENDING_STATE] = self._state
```

Comment:
We're calling the pre and post arming states pending here. That seems inconsistent.

Reply:
thanks for the remark, just updated that to make it more clear